localization/languages/cs-CZ.json

```diff
@@ -0,0 +1,152 @@
+{
+    "name": "Česky",
+    "identifier": "cs-CZ",
```

Comment:
Based on the language list (https://electron.atom.io/docs/api/locales/) I think the correct language code is just "cs". Are you seeing something different on your OS?

Reply:
That's good point. I don't know, what is correct.
My `locale | grep LANG=` returns **LANG=cs_CZ.UTF-8**.

Reply:
But `locale | grep LANGUAGE=` returns **LANGUAGE=cs:en_GB**.
Ok then, let's use only **cs**.

Reply:
Check [this](http://www.localeplanet.com/icu/cs-CZ/index.html).
> Java Locale | cs-CZ

Reply:
What do you get if you open the developer tools in Min and type ```remote.app.getLocale()```?

Reply:
> What do you get if you open...

Only `cs`.